frontend/src/apollo/update/conceptLink.js

```diff
@@ -5,6 +5,33 @@ import * as objectRecursion from '../../lib/objectRecursion'
 const includedIn = (set, object) =>
   set.map(p => p.id).includes(object.id)
 
+const updateConceptLinkUpdate = () => 
+  (store, response) => {
+    try {
+      const updatedConceptLink = response.data.updateConceptLink
+      const dataInStore = store.readQuery({
+        query: LINKS_IN_COURSE,
+        variables: { courseId: updatedConceptLink.to.course.id }
+      })
+      const dataInStoreCopy = { ...dataInStore }
+      const concept = dataInStoreCopy.linksInCourse.concepts
+        .find(concept => concept.id === updatedConceptLink.to.id)
+      
+      if (concept && !includedIn(concept.linksToConcept, updatedConceptLink)) {
```

Comment:
this `includedIn` call seems a bit weird, why does it matter whether or not the concept link is in the array?

Reply:
It was meant for debugging purposes in case the cache would not update.